### ISSUE TITLE
Finetuning of Kaleido-BERT for Fashion Captioning

### ISSUE THREAD

**Issue by gourango01** [mczhuge/Kaleido-BERT#6]
Thanks for sharing this interesting work. 
Would you please share how "Kaleido-BERT" has been fine-tuned on captioning task? 
Have you used separate decoder for generation or "Kaleido-BERT"  encoder only?

**Comment by mczhuge** [comment]
Encoder only, and we use a very simple method: [Mask] all then predict them. 
You can easily defeat the existing result if trying to use a useful decoder.

**Comment by gourango01** [comment]
**_During the fine-tuning on the image captioning task_**, Did you use any pre-training task (for e.g., AKPM, TIM and AMLM)  along with the fashion captioning task i.e., given an image ( **_i.e., sequence of image patches generated by "Kaleido Patch Generator"_**) predict the corresponding caption?
